templates/kops-cluster.tpl.yaml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: kops/v1alpha2
+kind: Cluster
+metadata:
+  name: ${name}
+spec:
+  api:
+    loadBalancer:
+      type: Public
+      idleTimeoutSeconds: 300
+  channel: stable
+  cloudProvider: aws
+  configBase: s3://${k8s_data_bucket}/${name}
+  etcdClusters:
+  - etcdMembers:
+${etcd_members_main}
+    name: main
+  - etcdMembers:
+${etcd_members_events}
+    name: events
+  kubernetesApiAccess:
+  - 0.0.0.0/0
+  kubernetesVersion: ${k8s_version}
+  masterInternalName: api.internal.${name}
+  masterPublicName: api.${name}
+  networkCIDR: ${vpc_cidr}
+  networkID: ${vpc_id}
+  networking:
+    cni: {}
```

Comment:
Because we want to use the plugins, no? Didn't you install the `calico` plugin yesterday?